src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -6704,6 +6720,705 @@ namespace HeatBalanceSurfaceManager {
         CalculateZoneMRT(ZoneToResimulate); // Update here so that the proper value of MRT is available to radiant systems
     }
 
+    void CalcHeatBalanceInsideSurf2CTFOnly(const int FirstZone,             // First zone to simulate
+                                           const int LastZone,              // Last zone to simulate
+                                           const std::vector<int> &IZSurfs, // Last zone to simulate
+                                           Optional_int_const ZoneToResimulate)
```

Comment:
`FirstZone` and `LastZone` arguments were originally here in the hope that this could be used for both modes - all zones or single zone resimulate - but it didn't work out that way. As it stands now, this function is never called with `ZoneToResimulate` - so maybe that should be dropped. I see the endline comment on `IZSurfs` is wrong, too. Will save these edits for later when addressing review comments.

So, as-is, this function could assume it always computes all zones and it could have no arguments at all. The hope was that it could work for the resimulate case, or be called for a single zone within a higher-level zone loop as part of future parallel flows. So, leaving the arguments there for now.